src/Encryption/Codec/EncryptionCodec.cs

```diff
@@ -8,20 +8,20 @@ namespace TemporalioSamples.Encryption.Codec;
 public sealed class EncryptionCodec : IPayloadCodec, IDisposable
 {
     public const string DefaultKeyID = "test-key-id";
-    public static readonly byte[] DefaultKey = System.Text.Encoding.ASCII.GetBytes("test-key-test-key-test-key-test!");
+    private static string DefaultKey = "test-key-test-key-test-key-test!";
     // Taken from other language samples for compatibility
     private const int NonceSize = 12;
     private const int TagSize = 16;
     private static readonly ByteString EncodingByteString = ByteString.CopyFromUtf8("binary/encrypted");
 
+    private readonly byte[] KeyBytes;
```

Comment:
```suggestion
    private readonly byte[] key;
```

No need to capitalize the private field and can just consider keys to always be byte arrays and not have/require a string form.